src/coredump/coredumpctl.c

```diff
@@ -569,9 +574,22 @@ static int print_info(FILE *file, sd_journal *j, bool need_space) {
         if (hostname)
                 fprintf(file, "      Hostname: %s\n", hostname);
 
-        if (filename)
-                fprintf(file, "       Storage: %s%s\n", filename,
-                        access(filename, R_OK) < 0 ? " (inaccessible)" : "");
+        if (filename) {
+                bool inacc = access(filename, R_OK) < 0;
+                bool trunc = streq_ptr(truncated, "yes");
```

Comment:
(btw, nitpick: i much prefer if we'd not mix variable declarations with function invocations... but doesn't matter much)